### DISCUSSION TITLE
GitHub standard is CONTRIBUTING.md instead of READMEdev.md, should we change it?

### DISCUSSION THREAD

**Discussion by datejada** [TulipaEnergy/TulipaEnergyModel.jl#404] [Questions]
@abelsiqueira, @clizbe what do you think? 
This is pure aesthetics, but it shows in the community standards that we don't have a contributing standard 😅 

<img width="731" alt="image" src="https://github.com/TulipaEnergy/TulipaEnergyModel.jl/assets/12887482/5f9a6bbc-3959-4bd0-bd92-2cee816b5865">

**Comment by abelsiqueira** [comment]
I like to separate CONTRIBUTING and README.dev.md (or something similar like DEVELOPING.md) in the following way:

- CONTRIBUTING.md tells how to get involved and what is expected
- README.dev.md tells how to setup your environment and how to do the technical things

That is how I proposed at the template: https://github.com/abelsiqueira/COPIERTemplate.jl/

Useful things for CONTRIBUTING:

- Forums and chatrooms
- How do we collaborate (issue vs discussion, fork -> branch -> PR -> squash (conceptually))
- Guidelines and standards
- Links to other useful parts

Useful things for README.dev.md

- Install VSCode extension
- Install pre-commit
- All the details for git
- How to build, test, build docs, etc.

These are all soft boundaries, of course, and the name `README.dev.md` is indeed niche, so in the end, follow your heart.

**Reply by clizbe** [reply]
I don't see a CONTRIBUTING in that template. Is it located somewhere else than next to README.dev?

**Reply by abelsiqueira** [reply]
No, I actually have them named something else as well: https://github.com/abelsiqueira/COPIERTemplate.jl/#what-are-all-these-files, below the file tree.
I wanted the contributing to be built in the documentation and to avoid duplication, so there is a docs/src/contributing.md instead.

**Comment by clizbe** [comment]
Based on @abelsiqueira's comment, we already have both in README.dev.md, which is also getting a bit long. Let's split it up!

**Comment by datejada** [comment]
Closed in #406
